### ISSUE TITLE
Wrong locale since SF update 4.3.1

### ISSUE THREAD

**Issue by rjwebdev** [lexik/LexikTranslationBundle#348]

**Comment by skytorner** [comment]
Hey @rjwebdev Do you still have this bug ? Because I updated my project too - same version as you now - and the translations won't work. 

Did you manage to get something working ?

Thanks

**Comment by rjwebdev** [comment]
I temporary use the fix made in the pull request from my previous comment untill the PR is merged and a new version of this package is released.

**Comment by skytorner** [comment]
Right, thanks for the indication, I didn't pay attention to the provided link ! 

I hope they will merge it and release it soon !
